BUILD.bazel

```diff
@@ -981,24 +938,263 @@ ray_cc_library(
+    hdrs = ["src/ray/raylet/scheduling/policy/affinity_with_bundle_scheduling_policy.h"],
+    deps = [
+        ":ray_common",
+        ":scheduling_policy",
+    ],
+)
+
+ray_cc_library(
+    name = "bundle_scheduling_policy",
+    srcs = ["src/ray/raylet/scheduling/policy/bundle_scheduling_policy.cc"],
+    hdrs = ["src/ray/raylet/scheduling/policy/bundle_scheduling_policy.h"],
+    deps = [
+        ":cluster_resource_manager",
+        ":ray_common",
+        ":scheduling_context",
+        ":scheduling_policy",
+        ":scorer",
+    ],
+)
+
+ray_cc_library(
+    name = "composite_scheduling_policy",
+    srcs = ["src/ray/raylet/scheduling/policy/composite_scheduling_policy.cc"],
+    hdrs = ["src/ray/raylet/scheduling/policy/composite_scheduling_policy.h"],
+    deps = [
+        ":affinity_with_bundle_scheduling_policy",
+        ":bundle_scheduling_policy",
+        ":cluster_resource_manager",
+        ":hybrid_scheduling_policy",
+        ":node_affinity_scheduling_policy",
+        ":node_label_scheduling_policy",
+        ":random_scheduling_policy",
+        ":spread_scheduling_policy",
+        "//src/ray/util",
+        "//src/ray/util:container_util",
+    ],
+)
+
+ray_cc_library(
+    name = "hybrid_scheduling_policy",
+    srcs = ["src/ray/raylet/scheduling/policy/hybrid_scheduling_policy.cc"],
+    hdrs = ["src/ray/raylet/scheduling/policy/hybrid_scheduling_policy.h"],
+    deps = [
+        ":scheduling_policy",
+        "//src/ray/util",
+        "//src/ray/util:container_util",
         "@com_google_absl//absl/random",
         "@com_google_absl//absl/random:bit_gen_ref",
-        "@com_google_absl//absl/strings",
-        "@com_google_googletest//:gtest",
-        "@io_opencensus_cpp//opencensus/exporters/stats/prometheus:prometheus_exporter",
-        "@io_opencensus_cpp//opencensus/stats",
-        "@io_opencensus_cpp//opencensus/tags",
+        "@com_google_googletest//:gtest_prod",
+    ],
+)
+
+ray_cc_library(
+    name = "node_affinity_scheduling_policy",
+    srcs = ["src/ray/raylet/scheduling/policy/node_affinity_scheduling_policy.cc"],
+    hdrs = ["src/ray/raylet/scheduling/policy/node_affinity_scheduling_policy.h"],
+    deps = [
+        ":hybrid_scheduling_policy",
+        ":scheduling_policy",
+    ],
+)
+
+ray_cc_library(
+    name = "node_label_scheduling_policy",
+    srcs = ["src/ray/raylet/scheduling/policy/node_label_scheduling_policy.cc"],
+    hdrs = ["src/ray/raylet/scheduling/policy/node_label_scheduling_policy.h"],
+    deps = [
+        ":scheduling_context",
+        ":scheduling_policy",
+    ],
+)
+
+ray_cc_library(
+    name = "random_scheduling_policy",
+    srcs = ["src/ray/raylet/scheduling/policy/random_scheduling_policy.cc"],
+    hdrs = ["src/ray/raylet/scheduling/policy/random_scheduling_policy.h"],
+    deps = [
+        ":scheduling_policy",
+        "//src/ray/util",
+        "//src/ray/util:container_util",
```

Comment:
`container_util` seems unused

Reply:
`const auto &node = map_find_or_die(nodes_, node_id);` still need this header.

Reply:
You're right, thank you for checking!